### PR TITLE
Added description for op call 0x2000xxxx

### PR DESCRIPTION
**Pull request by Athkore**
Signed-off-by: Andreas Kinnunen <kinnu@kth.se>

### CODE REVIEW

**Comment by NicholasMoser** [comment]
It is definitely true that this opcode is called on a new round or status reset, but more specifically, it seems to call `chr_init` on the character provided to the opcode. For example, I see it called three times when doing a 1v1 match of Naruto vs Sasuke (the third call is for Naruto's Partner Character clone). Do you mind updating the info to reflect this?

Maybe reword it to mention that it specifically calls `chr_init` on the opcode provided character.

image

**Comment by Athkore** [comment]
I added information that chr_init is called after stats have been reset, and that each character and partner character calls the op-code.

**Review by NicholasMoser — APPROVED** [review]
